chapters/equations.tex

```diff
@@ -390,16 +396,12 @@ \subsubsection{Application of the Single-assignment Rule to When-Equations}\labe
 end DoubleWhenConflict;
 \end{lstlisting}
 
-One way to resolve the conflict would be to give one of the two
-when-equations higher priority. This is possible by rewriting the
-when-equation using \lstinline!elsewhen!, as in the \lstinline!WhenPriority! model
-below or using the statement version of the when-construct, see \cref{when-statements}.
+One way to resolve the conflict would be to give one of the two \lstinline!when!-equations higher priority.
+This is possible by rewriting the \lstinline!when!-equation using \lstinline!elsewhen!, as in the \lstinline!WhenPriority! model below or using the statement version of the \lstinline!when!-construct, see \cref{when-statements}.
 \end{nonnormative}
 
 \item
-  When-equations involving elsewhen-parts can be used to resolve
-  assignment conflicts since the first of the when/elsewhen parts are
-  given higher priority than later ones:
+  \lstinline!when!-equations involving elsewhen-parts can be used to resolve assignment conflicts since the first of the when/elsewhen parts are given higher priority than later ones:
```

Comment:
```suggestion
  A \lstinline!when!-equation involving elsewhen-parts can be used to resolve assignment conflicts since the first of the when/elsewhen parts are given higher priority than later ones:
```